openapi_schema_to_json_schema/to_jsonschema.py

```diff
@@ -111,9 +111,14 @@ def convertProperties(properties, options):
 
 
 def validateType(ttype):
-    validTypes = ['integer', 'number', 'string', 'boolean', 'object', 'array']
+    validTypes = [
+        'integer', 'number', 'string', 'boolean', 'object', 'array', 'null'
+    ]
```

Comment:
I may be confused, but this should be the list of valid types in the OpenAPI 3 schema object and `null` is not allowed: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types

Reply:
Ahh i see. I thought this was checking the json schema types. 
The intent behind adding this check is, the convert process fails converting for this particular input schema: https://gist.github.com/riddhi89/4e43431ce400137465cd90dc88e2d5e5#file-gistfile1-txt-L63-L78.
Since the `convertschema` is being called recursively, the property conversion has already taken place in this case.

The failure seen for the above schema:
```
py36/lib/python3.6/site-packages/openapi_schema_to_json_schema/to_jsonschema.py", line 117, in validateType
    raise InvalidTypeError('Type "%s" is not a valid type' % ttype)
openapi_schema_to_json_schema.to_jsonschema.InvalidTypeError: Type "['string', 'null']" is not a valid type
```